src/async_jobs.ts

```diff
@@ -42,6 +62,25 @@ async function slackInteractivityHandler(
     );
   }
 
+  if (payload.type === 'shortcut') {
+    const { viewId } = interactivityMetadata;
+    if (!viewId) {
+      throw new Error(
+        'slackInteractivityHandler called for message_action without viewId'
+      );
+    }
+    switch (payload.callback_id) {
+      case 'show_needs_attention': {
+        await SlackInteractionHandler.receiveShowNeedsAttention({
```

Comment:
can you rename to `handleShowNeedsAttention`? Elsewhere I use `receive` to mean receive the command but don't execute it (ask for confirmation), and `handle` is to do the actual execution logic